docs/usage.md

```diff
@@ -58,6 +58,34 @@ Pipeline settings can be provided in a `yaml` or `json` file via `-params-file <
 
 Do not use `-c <file>` to specify parameters as this will result in errors. Custom config files specified with `-c` must only be used for [tuning process resource specifications](https://nf-co.re/docs/usage/configuration#tuning-workflow-resources), other infrastructural tweaks (such as output directories), or module arguments (args).
 
+### Lock
+
+The lock transformation may be run as follows:
+
+```bash
+nextflow run phac-nml/metadatatransformation -profile singularity -r main -latest --input assets/samplesheet.csv --outdir results --transformation lock
+```
+
+You may wish to specify the `--metadata_1_header` through `--metadata_8_header` parameters to ensure the metadata is named as desired:
+
+```bash
+nextflow run phac-nml/metadatatransformation -profile singularity -r main -latest --input assets/samplesheet.csv --outdir results --transformation lock --metadata_1_header country --metadata_2_header outbreak
+```
+
+### Age
+
+The calculate age transformation may be run as follows:
```

Comment:
Would it be good to specify that `metadata_1` and `metadata_2` are used as the `dob` and `collection date` respectively for the calculation?

Reply:
Good idea. Added in d11ab2e.